gameofgame.py

```diff
@@ -27,20 +28,29 @@ def check_guess(guess, country):
     if guess.lower() == capital.lower():
         return True
     else:
-        return False
+        return False   
+# defining main game logic
+
+
+def main():
```

Comment:
The `main()` is empty now, as the code block below has issues with indentation (4 spaces).